### DISCUSSION TITLE
Welcome to SuperYOLO Discussions!

### DISCUSSION THREAD

**Discussion by icey-zhang** [icey-zhang/SuperYOLO#117] [Announcements]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by icey-zhang** [comment]
我们正在使用讨论作为与社区其他成员联系的地方。我们希望您: 
 
问一些你想知道的问题。 
分享想法。 
与其他社区成员交流。 
欢迎他人，思想开放。记住，这是一个社区 
一起构建💪。 
要开始，请在下方留言介绍你自己，并告诉我们你在这个社区做了什么。

**Comment by lansimtech** [comment]
Thank you for your very useful project. May I ask you a question regarding data processing? As I understand, the VEDAI dataset you are experimenting with in SuperYOLO contains only about 1,100 high-resolution images. So, if I want to test your project with a dataset of around 30,000 high-resolution images, how should I handle it to reduce training time?
I look forward to your response.
Thank you very much.
